README.md

```diff
@@ -23,89 +23,36 @@ Storage Backends
-        type="org.plos.repo.service.ObjectStore"
-        factory="org.plos.repo.config.FileSystemStoreFactory"
-        dataDirectory="/path/to/data/directory"
-        reproxyBaseUrl = "http://localhost/objdata/" />
-
-Now you should be able to make object GET requests and ask for reproxied URLs.
-        
+Environment variables are used for the configuration of the storage backend.
 
-If you are using **Amazon S3** as the object store, set your access key and secret key.
+If you wish to use **MogileFS**, set the `MOGILE_TRACKERS` environment variable. For example:
+
+```
+MOGILE_TRACKERS=localhost:7001,otherhost:7001 mvn cargo:run
+```
 
-    <Resource name="repo/objectStore"
-        type="org.plos.repo.service.S3StoreService"
-        factory="org.plos.repo.config.S3StoreFactory"
-        awsAccessKey="abc"
-        awsSecretKey="def" />
-        
-For testing purposes there is also an **InMemoryFileStore** which you can simply use like so:
+If you are storing data on a **local filesystem**, set the 'DATA_DIRECTORY' environment variable accordingly. When you create the directory on your filesystem make sure it is owned by 'tomcat'.
 
-    <Resource name="repo/objectStore"
-        type="org.plos.repo.service.ObjectStore"
-        factory="org.plos.repo.config.InMemoryStoreFactory" />
+```
+DATA_DIRECTORY=/path/to/data/directory mvn cargo:run
+```        
 
+If you are using **Amazon S3** as the object store, set your access key and secret key.
 
+```
+AWS_ACCESS_KEY_ID=FOO AWS_SECRET_ACCESS_KEY= mvn cargo:run
+```
 
 Database Backends
 -----------------
 
-The service supports HSQLDB and MySQL. You will need to place one of the following Resources in Tomcat's context.xml :
-
-If you are using **MySQL**, you need to manually create a database (I called it 'repo' in the example below but call it what you want) and create a user with granted write permissions. 
-
-    <Resource name="jdbc/repoDB"
-        auth="Container"
-        type="javax.sql.DataSource"
-        factory="org.apache.tomcat.jdbc.pool.DataSourceFactory"
-        validationQuery="SELECT 1"
-        testOnBorrow="true"
-        driverClassName="com.mysql.jdbc.Driver"
-        username="root"
-        password=""
-        url="jdbc:mysql://localhost:3306/repo" />
-
-**HSQLDB** can support file and in memory databases. Set the 'url' to your configuration.
-
-    <Resource name="jdbc/repoDB"
-        auth="Container"
-        type="javax.sql.DataSource"
-        validationQuery="SELECT 1 FROM INFORMATION_SCHEMA.SYSTEM_USERS"
-        driverClassName="org.hsqldb.jdbc.JDBCDriver"
-        username=""
-        password=""
-        url="jdbc:hsqldb:file:/tmp/repo-hsqldb;shutdown=true" />
+The service supports MySQL. You will need to place one of the following Resources in Tomcat's context.xml :
+
+To configure the database, use the `DATABASE_URL` environment variable.
+
+```
+DATABASE_URL="jdbc:mysql://localhost:3306/repo?user=root&password=password" mvn cargo:run
```

Comment:
`DATABASE_URL="jdbc:mysql://root:password@localhost:3306/repo" mvn cargo:run`

works as well (just FYI)

Reply:
Really? I thought I tried that and I kept getting errors. That's awesome.